falcon/response.py

```diff
@@ -564,6 +564,23 @@ def add_link(self, target, rel, title=None, title_star=None,
         """,
         lambda v: ', '.join(v))
 
+    accept_ranges = header_property(
+        'Accept-Ranges',
+        """
+        Sets the Accept-Ranges header.
```

Comment:
Would you mind removing the newline after the triple-quote to be more consistent with the other property definitions? I.e.:

```py
"""Sets the Accept-Ranges header.

...

"""
```